content/docs/instrumenting/exporters.md

```diff
@@ -158,6 +158,7 @@ wide variety of JVM-based applications, for example [Kafka](http://kafka.apache.
    * [Munin exporter](https://github.com/pvdh/munin_exporter)
    * [Nagios / Naemon exporter](https://github.com/Griesbacher/Iapetos)
    * [New Relic exporter](https://github.com/mrf/newrelic_exporter)
+   * [nftables exporter](https://github.com/Intrinsec/nftables_exporter)
```

Comment:
Firewalls are not monitoring systems.

Reply:
> This looks like there might be some overlap with the node exporter, but it seems that it also requires privs - @SuperQ?

Hello, yes I started this as an collector in node_exporter for iptables and was rejected as it required some additional capabilities.

> It'd be better to fail the scrape if there's a problem, rather than giving the user an extra metric to have to check.

I made it return nothing when it fail like for example for missing capabilities, after the initial review back then is it not the best practice ?

> Firewalls are not monitoring systems.
Yes my  bad, should it be put in Miscellaneous ?